src/masternodes/rpc_evm.cpp

```diff
@@ -144,42 +149,81 @@ UniValue evmtx(const JSONRPCRequest& request) {
     return send(MakeTransactionRef(std::move(rawTx)), optAuthTx)->GetHash().ToString();
 }
 
+UniValue handleMapBlockNumberEVMToDVMRequest(const std::string &input) {
```

Comment:
Maybe we can change this function to `mapBlockNumberEVMToDVMRequest`? Don't think we have any other functions called `handle*`.

Reply:
`handle` is for things that return actually validates and returns JSON / univalue errors. If it's map, it should only have returned been a pure map fn, which here it's not. Should be there in another discussion thread.